Dockerfile

```diff
@@ -95,6 +100,25 @@ RUN sip-install \
 WORKDIR /output/
 RUN find /upstream/ -name \*.pyi -exec cp {} . \;
 
+################################################################################
+# PyQt5-SIP stubs
+################################################################################
+
+FROM build-dep AS sip
+
+# Reuse arguments from previous build scope
+ARG SIP_VERSION
+ARG SIP_ABI_VERSION
+
+# Install SIP including stubs
+# TODO: Find way to build only stubs. This takes way too long
+WORKDIR /upstream/
+RUN pip install --no-deps --target build sip==${SIP_VERSION}
```

Comment:
We might want to make this download consistent with the other build layers. 

I think when I wrote the rest of the Dockerfile, I downloaded directly from PyPI's CDN to avoid the pip install, but in hindsight that may have been overkill. I'm not opposed to making every layer just use pip.

Reply:
Ok, so now that I look the regular PyQt5 package provides its own `.pyi` files.  Why are we building ourselves?  (regardless of how we get the sdist)

Reply:
They didn't used to be: https://github.com/stlehmann/PyQt5-stubs/issues/40

Reply:
I vote we don't mix switching away from building into this right now.  Maybe?  Though I do like consistency.  Turns out the `.pyi` are right in the sdist so I'll switch over to the download/extract/copy structure.  We can revisit not-building separately.

Reply:
Is the update what you were looking for?

Reply:
I'm not familiar enough with the differences between `find -name` and `find -wholename`, but assuming you had a reason to go with the latter, lgtm

Reply:
Yeah, I knew `-name` was wrong because I had the parent directory in the path but I ran it anyways and it told me to `-wholename`.  At least the present sdist provides two ABIs so we need to pick one over the other.